src/handlers/interventions-processor-handler.ts

```diff
@@ -93,14 +103,14 @@ async function processSQSRecord(itemFailures: SQSBatchItemFailure[], record: SQS
  *
  * @param recordBody - the parsed body of the sqs record
  */
-function isTimestampNotInFuture(recordBody: TxMAEvent): boolean {
+function isTimestampInFuture(recordBody: TxMAEvent): boolean {
   const now = getCurrentTimestamp().milliseconds;
   if (now < (recordBody.event_timestamp_ms ?? recordBody.timestamp * 1000)) {
     logger.debug(`Timestamp is in the future (sec): ${recordBody.timestamp}.`);
```

Comment:
logger.warn(`Timestamp is in the future (sec): ${recordBody.event_timestamp_ms ?? recordBody.timestamp * 1000}.`);